library.properties

```
@@ -1,5 +1,5 @@
 name=RTCZero
-version=1.6.0
+version=2.0.0
```

Does this PR introduce a breaking change to the API?

---

It does not.  I initially expected it to but failed to revisit this value after ironing things out.
```suggestion
version=1.7.0
```